index.html

```diff
@@ -70,7 +74,7 @@ <h2 id="priority"><i>Aperture priority</i></h2>
       </tr>
     </thead>
     <tbody>
-      <tr>
```

Comment:
The only actual change here is that there was an extra space on the end of this line. Those are harmless, but I have my editor set to automatically remove extra spaces at the end whenever it saves.